assets/xml/objects/object_rat.xml

```diff
@@ -1,28 +1,38 @@
 ﻿<Root>
+    <!-- Assets for Real Bombchus -->
```

Comment:
Where does the name "Real Bombchu" come from, I would picture it more as something like "Enemy Bombchu" perhaps?

Reply:
Straight from the game itself:

![NKZE01_2022-07-24_17-05-25](https://user-images.githubusercontent.com/12245827/180671590-778f1ec8-1e58-46c6-8ec1-1ae20741b873.png)